src/ext/oblivc/obliv_bits.c

```diff
@@ -1842,6 +681,7 @@ void feedOblivInputs(OblivInputs* spec, size_t count, int party)
     { __obliv_c__##ot rv; \
       OblivInputs spec; \
       setupObliv##tname(&spec,&rv,v); \
+      spec.src_f = v; \
```

Comment:
Thanks for that detailed response. Again, I apologize as it's been a while since I wrote that code but I am fairly sure that this actually does work. The float debugging protocol is not what I used for testing -- the tests are in the ```test/``` folder. Specifically there is ```test/testFloat``` that tests all of the floating point operations and there is ```test/linreg``` that runs linear regression. Both of these use Yao's protocol, not the float debugging protocol. In order to make sure that it does work, I just ran some additional code (with Yao's protocol) that uses a mix of ```obliv int```s and ```obliv float```s and it executed without issue. Again, I don't recall why I wrote it this way but I will look into it to find the reason.